src/encode.cpp

```diff
@@ -3662,7 +3636,6 @@ static void encodeBlock(
     ENCODE(st, op, mlir::linalg::FillOp, encodeMemWriteOps);
     ENCODE(st, op, mlir::linalg::GenericOp, encodeMemWriteOps);
     ENCODE(st, op, mlir::linalg::IndexOp, encodeMemWriteOps);
-    ENCODE(st, op, mlir::linalg::InitTensorOp, encodeMemWriteOps);
```

Comment:
It is slightly sad that this `ENCODE` is permanently being removed.
Do you think it is feasible to add
```
ENCODE(st, op, mlir::tensor::EmptyOp, encodeMemWriteOps);
```
again? It might require a very small change to the existing `encodeOp` for `linalg::InitTensorOp`.